database/migrations/2023_10_05_103137_create_address_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('addresses', function (Blueprint $table) {
+            $table->id();
+            $table->foreignId('user_id')->constrained()->cascadeOnDelete();
+            $table->string('name');
+            $table->foreignId('region_id')->constrained()->cascadeOnDelete();
+            $table->string('street');
+            $table->integer('building_number');
+            $table->integer('apartment_number');
```

Comment:
make the additional information, apartment number nullable
The user may not provide his apartment number (he wants it to be delivered to his building and taken it also we may have a delivery to another store)